UnityProject/Assets/Scripts/Objects/StatusDisplay.cs

```diff
@@ -290,8 +299,14 @@ private IEnumerator BlinkText()
 			this.StartCoroutine(BlinkText(), ref blinkHandle);
 		}
 
+		private void UpdateTextColor(Color oldValue, Color newValue)
+		{
+			textField.color = centComm.CurrentAlertLevel == CentComm.AlertLevel.Red ? redAlertTextColor : normalTextColor;
```

Comment:
This is in the client though how do they know what CurrentAlertLevel is?

 That's why you use the Color oldValue, Color newValue) on client